packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -58,6 +56,14 @@ const optionalBuildDirectoryName = 'build';
 const cfnTemplateGlobPattern = '*template*.+(yaml|yml|json)';
 const parametersJson = 'parameters.json';
 
+
+const deploymentInProgressErrorMessage = (context: $TSContext) => {
+  context.print.error('A deployment in progress.');
+  context.print.error('If you believe the prior deployment was aborted, run:');
```

Comment:
```suggestion
  context.print.error('To abort and rollback the deployment, run:');
```

Reply:
Abort would mean the cli lost connection mid deployment. Maybe 
`If the prior rollback was aborted...`?